bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/DynamicSequenceCellRenderer.java

```diff
@@ -17,28 +14,24 @@
 final class DynamicSequenceCellRenderer extends SequenceCellRenderer {
     private static final Color VERY_LIGHT_GREY = new Color(247, 243, 239);
 
-    private static final Set<ItemKey> FIXED_ITEMS = new HashSet<ItemKey>();
-
-    static {
-        for (ItemKey key : DynamicSequenceTableModel.SORT_ORDER) {
-            FIXED_ITEMS.add(key);
-        }
-    }
+    private static final Set<ItemKey> FIXED_ITEMS = new HashSet<ItemKey>() {{
+        Collections.addAll(this, DynamicSequenceTableModel.SORT_ORDER);
+    }};
 
     private DynamicSequenceTableModel _model;
 
-    DynamicSequenceCellRenderer(DynamicSequenceTableModel model) {
+    DynamicSequenceCellRenderer(final DynamicSequenceTableModel model) {
         _model  = model;
     }
 
-    protected Color lookupTextColor(int row, int column) {
+    protected Color lookupTextColor(final int row, final int column) {
         return _model.matchesNodeId(row) ?
                 super.lookupTextColor(row, column) :
                 Color.LIGHT_GRAY;
     }
 
-    protected Color lookupColor(int row, int column) {
-        ItemKey key = _model.getItemKeyAt(column);
+    protected Color lookupColor(final int row, final int column) {
+        final ItemKey key = _model.getItemKeyAt(column);
         Color color = VERY_LIGHT_GREY;
```

Comment:
This could also be made final with a ternary expression or by breaking the `if` statement into an `if`...`else` and not assigning to `color` initially.